test_runner/regress/test_pageserver_generations.py

```diff
@@ -703,7 +703,6 @@ def test_multi_attach(
     workload.validate(pageservers[2].id)
 
 
-@pytest.mark.skip(reason="To be enabled after release with new local path style")
```

Comment:
Just making sure you're not slipping this through accidentally.

Reply:
oops, that should have been reinstated in the last commit.  bbd7e50e3